example/sdvc.postman_collection.json

```diff
@@ -193,6 +193,55 @@
 					},
 					"response": []
 				},
+				{
```

Comment:
it may be better to leave this collection as is and add cameo specific tests to the cameo collection json (there's one already for testing mounts) - i've wanted to separate out the tests into more logical collections but haven't made an issue yet. 
To fix this collection so it's not making a cameo project - change the project posts json to have projectType: default (i think leaving it out would also work)

Reply:
Changed the sdvc collection to use 'default' projects instead of 'cameo' to avoid the errors.  Added a test for the default elements to the cameo collection